relayer/relays/beacon/header/header.go

```diff
@@ -357,7 +383,12 @@ func (h *Header) populateClosestCheckpoint(slot uint64) (cache.Proof, error) {
 		}
 		err := h.populateFinalizedCheckpoint(checkpointSlot)
 		if err != nil {
-			return cache.Proof{}, fmt.Errorf("populate closest checkpoint: %w", err)
+			// 4513792 finalized slot
+			attestedHeaderSlot := checkpointSlot + (32 * 2.5)
```

Comment:
Just curious about the magic number here.

Reply:
Just hardcoded for now to test, I'll change it. 😄